tools/koch/koch.nim

```diff
@@ -24,6 +24,7 @@ import std/[json, os, strutils, parseopt, osproc, times]
   # Using `std/os` instead of `os` to fail early if config isn't set up properly.
   # If this fails with: `Error: cannot open file: std/os`, see
   # https://github.com/nim-lang/Nim/pull/14291 for explanation + how to fix.
+import ../compiler/utils/nodejs
```

Comment:
Maybe it's better to put the tests in a `ci` category run by testament, as testament already imports compiler/utils/nodejs anyways..

Reply:
IMO testament `ci` category would be better, it keeps `koch` that much slimmer.

Reply:
I put it into the `tests/tools` category, since the javascript file also sits under `tools`.